custom-login/src/main/resources/templates/home.html

```diff
@@ -32,7 +32,7 @@ <h2>Custom Login + Spring Boot Example</h2>
       </div>
 
       <div th:if="${#authorization.expression('isAuthenticated()')}" class="text container">
-        <p>Welcome home, <span th:text="${#authentication.name}">Joe Coder</span>!</p>
+        <p>Welcome home, <span th:text="${#authentication.principal.attributes['name']}">Joe Coder</span>!</p>
```

Comment:
What changed here?
Our other samples use the same pattern:
https://github.com/okta/samples-java-spring/blob/master/okta-hosted-login/src/main/resources/templates/userProfile.html#L28

Reply:
Background of this change: https://github.com/okta/samples-java-spring/issues/63

Reply:
☺️ sorry about that, I didn't remember that one :D